common/taskana-common/src/main/java/pro/taskana/common/api/WorkingTimeCalculator.java

```diff
@@ -6,47 +6,161 @@
 import java.time.LocalDate;
```

Comment:
As discussed:
* Move WorkingDaysToDaysConverter.java, WorkingTimeCalculator.java and WorkingTimeSchedule.java to their corresponding internal package.
* Create Interface for WorkingTimeCalculator.java

Reply:
Is it really ok to move WorkingDaysToDaysConverter? It provides public methods like:

- isWorkingDay(LocalDate)
- isHoliday(LocalDate)
- hasWorkingDaysInBetween(Instant,Instant)

But I agree that we could remove all "calculation" methods:
- addWorkingDaysToInstant(Instant, Duration)
- subtractWorkingDaysFromInstant(Instant, Duration)
-

Reply:
Should we "mask" those methods in the WorkingTimeCalculator and document that they have been moved?

Reply:
done